plugin/pom.xml

```diff
@@ -181,9 +182,9 @@
       <artifactId>apache-httpcomponents-client-4-api</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.google.code.gson</groupId>
-      <artifactId>gson</artifactId>
-      <version>2.10.1</version>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>gson-api</artifactId>
+      <version>2.10.1-15.v0d99f670e0a_7</version>
```

Comment:
Don't include version if part of bom

Reply:
If I remove the version I get the following error:

```
[ERROR]   The project io.jenkins.plugins:warnings-ng:10.7.0-SNAPSHOT (C:\Users\tiwar\open-source\warnings-ng-plugin\plugin\pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for io.jenkins.plugins:gson-api:jar is missing. @ io.jenkins.plugins:warnings-ng:${revision}${changelist}, 
```

Reply:
So use a bom version that include gson-api. Is part of bom since something like 1 month